src/Components/Web/src/BindInputElementAttribute.cs

```diff
@@ -25,7 +25,7 @@ public sealed class BindInputElementAttribute : Attribute
         /// <param name="format">
         /// An optional format to use when converting values. 
         /// </param>
-        public BindInputElementAttribute(string type, string suffix, string valueAttribute, string changeAttribute, bool isInvariantCulture, string format)
+        public BindInputElementAttribute(string? type, string? suffix, string? valueAttribute, string? changeAttribute, bool isInvariantCulture, string? format)
```

Comment:
What does it mean for the `type` to be null? Isn't type required? Or does it default to "text"?

In cases like these, we should consider applying defaults to the properties when we receive null-values and avoid having the code that uses these things from having to deal with null values.

Reply:
https://github.com/dotnet/aspnetcore/blob/602cb1dea59751a1297d674cddf7f0729c23e6ca/src/Components/Web/src/Web/BindAttributes.cs#L16